opscenter/README.md

```diff
@@ -76,14 +76,33 @@ You'll also need to specify SMTP settings if you wish to send email. Finally, yo
 
 Once you've made your changes, save the file as `opscenter.yaml` in this directory.
 
-
 Provisioning an OpsCenter
 ---
 Included in this directory is configuration to provision a Vagrant VM, as well as an AWS instance to run your OpsCenter. Once you have created the required files above, you can provision one of your choices as follows:
 
 - AWS: `make aws KEY_PAIR=<aws key name> TOKEN=<api token>`
 - Vagrant: `make vagrant TOKEN=<api token>`
 
+Manually provisioning
```

Comment:
Sorry, I meant manually installing, assuming users have their own provisioning step, wanted to show the steps that we actually doing on the installed box.